test-app/package.json

```diff
@@ -28,7 +28,7 @@
   "devDependencies": {
     "@ember/optional-features": "^2.0.0",
     "@ember/test-helpers": "^2.8.1",
-    "@embroider/test-setup": "^1.7.1",
+    "@embroider/test-setup": "^2.1.1",
```

Comment:
this upgrade fixes the issue that embroiderOptimized/safe was handling before: https://github.com/embroider-build/embroider/blob/main/CHANGELOG.md#embroidertest-setup-202---210